File: core/src/main/java/util/url/UrlUtils.java
package util.url;

/**
 * Created on 2019/8/4.
 *
 * @author 郑少鹏
 * @desc UrlUtils
 */
public class UrlUtils {
    /**
     * Whether the url is a local one.
     *
     * @param url String
     * @return boolean
     */
    public static boolean areLocal(String url) {
        return ((null != url) && !url.startsWith("http://") && !url.startsWith("https://"));
    }
}

File: core/src/main/java/util/url/UrlUtils.kt
package util.url

/**
 * Created on 2026/4/8.
 * @author 郑少鹏
 * @desc UrlUtils
 */
class UrlUtils {
    companion object {
        /**
         * Whether the url is a local one.
         *
         * @param url String
         * @return boolean
         */
        fun areLocal(url: String?): Boolean = url?.let {
            !url.startsWith("http://") && !url.startsWith("https://")
        } ?: false
    }
}